### DISCUSSION TITLE
Display releases by forks

### DISCUSSION THREAD

**Discussion by nisbet-hubbard** [community/community#137666] [Repositories]
### Select Topic Area

Product Feedback

### Body

It’d be really nice if besides the number of stars and PRs, displaying the active forks of a repo will also inform us whether a fork has made its own releases and the latest release number. This way, one can tell at a glance if a fork has committed itself to offering an improved product worth exploring.

**Comment by 23Benji** [comment]
That’s a great suggestion! Adding information about active forks, their releases, and the latest release number would definitely help users evaluate if a fork is worth checking out. It would make it easier to identify well-maintained forks and their contributions at a glance. Thanks for sharing your idea!

**Comment by lukaseder** [comment]
Perhaps this could be solved by sorting releases by semantic version number rather than by date, and then possibly filter on major/minor release streams.
